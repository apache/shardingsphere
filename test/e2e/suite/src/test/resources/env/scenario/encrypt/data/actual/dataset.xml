<!--
  ~ Licensed to the Apache Software Foundation (ASF) under one or more
  ~ contributor license agreements.  See the NOTICE file distributed with
  ~ this work for additional information regarding copyright ownership.
  ~ The ASF licenses this file to You under the Apache License, Version 2.0
  ~ (the "License"); you may not use this file except in compliance with
  ~ the License.  You may obtain a copy of the License at
  ~
  ~     http://www.apache.org/licenses/LICENSE-2.0
  ~
  ~ Unless required by applicable law or agreed to in writing, software
  ~ distributed under the License is distributed on an "AS IS" BASIS,
  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
  -->

<dataset>
    <metadata data-nodes="encrypt.t_user">
        <column name="user_id" type="numeric" />
        <column name="user_name_cipher" type="varchar" />
        <column name="user_name_like" type="varchar" />
        <column name="user_name_plain" type="varchar" />
        <column name="password_cipher" type="varchar" />
        <column name="email_cipher" type="varchar" />
        <column name="telephone_cipher" type="varchar" />
        <column name="telephone_like" type="varchar" />
        <column name="telephone_plain" type="varchar" />
        <column name="creation_date" type="datetime" />
    </metadata>
    <metadata data-nodes="encrypt.t_user_item">
        <column name="item_id" type="numeric" />
        <column name="user_id" type="numeric" />
        <column name="status" type="varchar" />
        <column name="creation_date" type="datetime" />
    </metadata>
    <metadata data-nodes="encrypt.t_single_table">
        <column name="single_id" type="numeric" />
        <column name="id" type="numeric" />
        <column name="status" type="varchar" />
    </metadata>
    <metadata data-nodes="encrypt.t_merchant">
        <column name="merchant_id" type="numeric" />
        <column name="country_id" type="numeric" />
        <column name="merchant_name" type="varchar" />
        <column name="business_code_cipher" type="varchar" />
        <column name="business_code_like" type="varchar" />
        <column name="business_code_plain" type="varchar" />
        <column name="telephone_cipher" type="varchar" />
        <column name="telephone_like" type="varchar" />
        <column name="telephone_plain" type="varchar" />
        <column name="creation_date" type="datetime" />
    </metadata>
    <row data-node="encrypt.t_single_table" values="1, 10, init" />
    <row data-node="encrypt.t_single_table" values="2, 11, init" />
    <row data-node="encrypt.t_single_table" values="3, 12, init" />
    <row data-node="encrypt.t_single_table" values="4, 13, init" />
    <row data-node="encrypt.t_user" values="10, sVq8Lmm+j6bZE5EKSilJEQ==, yi`mht`m, zhangsan, aQol0b6th65d0aXe+zFPsQ==, WM0fHOH91JNWnHTkiqBdyNmzk4uJ7CCz4mB1va9Ya1M=, kLjLJIMnfyHT2nA+viaoaQ==, 01454589811, 12345678900,  2017-08-08" />
    <row data-node="encrypt.t_user" values="11, fQ7IzBxKVuNHtUF6h6WSBg==, mhth, lisi, wuhmEKgdgrWQYt+Ev0hgGA==, svATu3uWv9KfiloWJeWx3A==, 0kDFxndQdzauFwL/wyCsNQ==, 01454589810, 12345678901,  2017-08-08" />
    <row data-node="encrypt.t_user" values="12, AQRWSlufQPog/b64YRhu6Q==, x`mhxt, wangwu, x7A+2jq9B6DSOSFtSOibdA==, nHJv9e6NiClIuGHOjHLvCAq2ZLhWcqfQ8/EQnIqMx+g=, a/SzSJLapt5iBXvF2c9ycw==, 01454589811, 12345678902,  2017-08-08" />
    <row data-node="encrypt.t_user" values="13, 5NqS4YvpT+mHBFqZOZ3QDA==, yi`pmht, zhaoliu, zi6b4xYRjjV+bBk2R4wB+w==, MLBZczLjriUXvg3aM5QPTxMJbLjNh8yeNrSNBek/VTw=, b6VVhG+F6ujG8IMUZJAIFg==, 01454589814, 12345678903,  2017-08-08" />
    <row data-node="encrypt.t_user" values="14, qeIY9od3u1KwhjihzLQUTQ==, yitph, zhuqi, 51UmlLAC+tUvdOAj8CjWfQ==, JCmeNdPyrKO5BW5zvhAA+g==, f995xinpZdKMVU5J5/yv3w==, 01454589815, 12345678904,  2017-08-08" />
    <row data-node="encrypt.t_user" values="15, VbNUtguwtpeGhHGnPJ3aXg==, mha`, liba, +3/5CVbqoKhg3sqznKTFFQ==, T+X+e3Q3+ZNIXXmg/80uxg==, GETj+S6DrO042E7NuBXLBQ==, 01454589814, 12345678905,  2017-08-08" />
    <row data-node="encrypt.t_user" values="16, U0/Ao/w1u7L5avR3fAH2Og==, x`mhiht, wangjiu, jFfFMYxv02DjaFRuAoCDGw==, RNW/KRq5HeL2YTfAdXSyARMJbLjNh8yeNrSNBek/VTw=, +lbvjJwO7VO4HUKc0Mw0NA==, 01454589815, 12345678906,  2017-08-08" />
    <row data-node="encrypt.t_user" values="17, zb1sgBigoMi7JPSoY4bAVw==, yite`, zhuda, VFIjocgjujJCJc6waWXqJA==, 1vF/ET3nBxt7T7vVfAndZQ==, wFvs5BH6OikgveBeTEBwsQ==, 01454589818, 12345678907,  2017-08-08" />
    <row data-node="encrypt.t_user" values="18, rJzNIrFEnx296kW+N1YmMw==, ttmdq, suner, LaODSKGyR7vZ1IvmBOe9vA==, 5u4GIQkJsWRmnJHWaHNSjg==, uwqm2O1Lv2tNTraJX1ym7Q==, 01454589819, 12345678908,  2017-08-08" />
    <row data-node="encrypt.t_user" values="19, qHwpQ9kteL8VX6iTUhNdbQ==, yiptt`m, zhousan, MyOShk4kjRnds7CZfU5NCw==, HmYCo7QBfJ2E0EvaGHBCOBMJbLjNh8yeNrSNBek/VTw=, YLNQuuUPMGA21nhKWPzzsg==, 01454589818, 12345678909,  2017-08-08" />
    <row data-node="encrypt.t_user" values="20, qCCmvf7OWRxbVbtLb0az1g==, upl, tom, fzdTMkzpBvgNYmKSQAp8Fg==, gOoP4Mf0P4ISOJp6A4sRmg==, l4xa4HwOfs/jusoJon9Wzw==, 01454589801, 12345678910,  2017-08-08" />
    <row data-node="encrypt.t_user" values="21, IYJ1COaRQ0gSjWMC/UAeMg==, lpad, kobe, 1uEDMeYh2jstbOf6kx/cqw==, tikMAFiQ37u2VgWqUT38Eg==, rGpr30UXfczXjCjdvPN+BA==, 01454589800, 12345678911,  2017-08-08" />
    <row data-node="encrypt.t_user" values="22, 7wvZZ7NVHgk6m1vB/sTC1Q==, idqqx, jerry, OirN3gvz9uBnrq88nfa1wQ==, T7K/Uz1O2m+3xvB0+c4nGQ==, 7+fCU+VbQZKgLJXZPTTegA==, 01454589801, 12345678912,  2017-08-08" />
    <row data-node="encrypt.t_user" values="23, SbVQWl8JbnxflCfGJ7KZdA==, i`ldt, james, hWVVYdkdTUTgm08haeq+tw==, Uk3ju6GteCD1qEHns5ZhKA==, DpnV86FZefwBRmIAVBh2gg==, 01454589804, 12345678913,  2017-08-08" />
    <row data-node="encrypt.t_user" values="24, fx7OfSAYqVpjNa7LoKhXvw==, x`ed, wade, N2W9ijAXNkBxhkvJiIwp0A==, lAAGItVLmb1H69++1MDrIA==, QrE62wAb8B+2cEPcs4Lm1Q==, 01454589805, 12345678914,  2017-08-08" />
    <row data-node="encrypt.t_user" values="25, wH3/LdWShD9aCb8eCIm3Tg==, qptd, rose, GDixtt6NzPOVv6H0dmov5g==, T1yfJSyVxumZUfkDnmUQxA==, iU+AsGczboCRfU+Zr7mcpw==, 01454589804, 12345678915,  2017-08-08" />
    <row data-node="encrypt.t_user" values="26, GgJQTndbxyBZ2tECS8SmqQ==, apti, bosh, gLgVFLFIyyKwdQCXaw78Ag==, O+JIn9XZ3yq6RnKElHuqlA==, kwYlbu9aF7ndvMTcj8QBSg==, 01454589805, 12345678916,  2017-08-08" />
    <row data-node="encrypt.t_user" values="27, lv8w8g32kuTXNvSUUypOig==, i`dl, jack, 8i0YH2mn6kXSyvBjM5p+Yg==, gqRoJF5S66SvBalc2RCo1A==, 2ob/3UYqRsZA5VdScnaWxQ==, 01454589808, 12345678917,  2017-08-08" />
    <row data-node="encrypt.t_user" values="28, P9YCbFvWCIhcS99KyKH2zA==, ipqe`m, jordan, PRrI4z4FrWwLvcHPx9g4og==, y8q31Jj4PFSyZHiLVIxKEQq2ZLhWcqfQ8/EQnIqMx+g=, kDF2za26uOerlNYWYHRT2Q==, 01454589809, 12345678918,  2017-08-08" />
    <row data-node="encrypt.t_user" values="29, 5wu9XvlJAVtjKijhxt6SQQ==, itmhd, julie, O4pgkLgz34N+C4bIUOQVnA==, UH7ihg16J61Np/EYMQnXIA==, z2hbJQD4dRkVVITNxAac5Q==, 01454589808, 12345678919,  2017-08-08" />
    <row data-node="encrypt.t_user_item" values="100000, 10, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="110000, 11, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="120000, 12, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="130000, 13, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="140000, 14, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="150000, 15, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="160000, 16, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="170000, 17, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="180000, 18, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="190000, 19, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="200000, 20, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="210000, 21, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="220000, 22, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="230000, 23, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="240000, 24, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="250000, 25, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="260000, 26, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="270000, 27, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="280000, 28, init, 2017-08-08" />
    <row data-node="encrypt.t_user_item" values="290000, 29, init, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="1, 86, tencent, k/LC4zsaZ+N6RgluwW+8BQ==, 95111110, 86000001, hRoqv9Jw8ZFM6Zdt3PL1Aw==, 95011111110, 86100000001, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="2, 86, haier, RINN9TpLonpAT8bAdJmKbA==, 95111111, 86000002, 9SCFkitrhrqandvIXg2U5Q==, 95011111111, 86100000002, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="3, 86, huawei, 3S6kkaTmrJNzCfuy07df4A==, 95111114, 86000003, L1okJVzt5ixaGjd1fUmujA==, 95011111114, 86100000003, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="4, 86, alibaba, KTWle6y9MyLX/G1uWqkB6Q==, 95111115, 86000004, JM3vuwFKlmUTM2wIxAx4kQ==, 95011111115, 86100000004, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="5, 86, lenovo, jYwQmxM2Hh/wZNFRfZZZog==, 95111114, 86000005, qD8jh5nV00thQ52QUAEMFQ==, 95011111114, 86100000005, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="6, 86, moutai, R9AAlR7vT5BttQCNUa9mFg==, 95111115, 86000006, FTritkoFVdFi7qbxQJbIKg==, 95011111115, 86100000006, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="7, 86, baidu, 6meW3pCZ3AllyhjcJLHfkg==, 95111118, 86000007, 6yJKW9FELJPUhWMD/hwa3g==, 95011111118, 86100000007, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="8, 86, xiaomi, 26tc7cYxmWSVjkCs3hBWfg==, 95111119, 86000008, rlaEwKNsNkpBTTJvX0cBWg==, 95011111119, 86100000008, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="9, 86, vivo, a6qv2URpZefcqeNwzYHYzw==, 95111118, 86000009, 4vSnvL4hUYAgyglex3Mo8g==, 95011111118, 86100000009, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="10, 86, oppo, Ly04f1inKr2V/KXAQbPY6g==, 95111101, 86000010, jNCHGPtcuha18Yxuz0hjjA==, 95011111101, 86100000010, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="11, 1, google, d93wyD7tAh6c0eV4JYgHDw==, 10111100, 01000011, T/8KO+rogDJlR3j0xxdXQQ==, 10011111100, 01100000011, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="12, 1, walmart, zTMndboJn5PRINho8p8vDw==, 10111101, 01000012, u+IOpGrCkTwAZYFDgRHYTg==, 10011111101, 01100000012, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="13, 1, amazon, OmMvy0O8jMH1cI5npzW84g==, 10111104, 01000013, 6xelCVyGaivIe70c5UKgJA==, 10011111104, 01100000013, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="14, 1, apple, aT6NnH1qsdpsLgOY0N1tKA==, 10111105, 01000014, hD+Wuog7jzaBzIyzRL97OA==, 10011111105, 01100000014, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="15, 1, microsoft, FP+W5tbInmmpOt0Nkmu8dw==, 10111104, 01000015, p4Esh1flQZFpApQKvfGkdQ==, 10011111104, 01100000015, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="16, 1, dell, vHvpwT5H2ahx8zrrenQpug==, 10111105, 01000016, LlJDQrh6CCKjRHYTwrJRVA==, 10011111105, 01100000016, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="17, 1, johnson, AbetbVO1CUdPjMv4XYlcaA==, 10111108, 01000017, dvljaVvLyAzW1Kcr4n8SvA==, 10011111108, 01100000017, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="18, 1, intel, PQ32nokZZTz9aFZvB7kwnw==, 10111109, 01000018, E4W6UfdyySk1GZRK5bkKWw==, 10011111109, 01100000018, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="19, 1, hp, uc1BIrytIfjEHdTa8xytlQ==, 10111108, 01000019, moteUYwjL/OtQphkcxsAAQ==, 10011111108, 01100000019, 2017-08-08" />
    <row data-node="encrypt.t_merchant" values="20, 1, tesla, lRucj/mOJODy3WgRpFS8vQ==, 10111111, 01000020, Blb7lRLTM7cL0f3HhrjtGw==, 10011111111, 01100000020, 2017-08-08" />
</dataset>
