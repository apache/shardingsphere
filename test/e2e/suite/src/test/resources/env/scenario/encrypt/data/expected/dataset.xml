<!--
  ~ Licensed to the Apache Software Foundation (ASF) under one or more
  ~ contributor license agreements.  See the NOTICE file distributed with
  ~ this work for additional information regarding copyright ownership.
  ~ The ASF licenses this file to You under the Apache License, Version 2.0
  ~ (the "License"); you may not use this file except in compliance with
  ~ the License.  You may obtain a copy of the License at
  ~
  ~     http://www.apache.org/licenses/LICENSE-2.0
  ~
  ~ Unless required by applicable law or agreed to in writing, software
  ~ distributed under the License is distributed on an "AS IS" BASIS,
  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
  -->

<dataset>
    <metadata data-nodes="expected_dataset.t_user">
        <column name="user_id" type="numeric" />
        <column name="address_id" type="numeric" />
        <column name="pwd" type="varchar" />
        <column name="status" type="varchar" />
    </metadata>
    <metadata data-nodes="expected_dataset.t_user_item">
        <column name="item_id" type="numeric" />
        <column name="user_id" type="numeric" />
        <column name="status" type="varchar" />
        <column name="creation_date" type="datetime" />
    </metadata>
    <metadata data-nodes="expected_dataset.t_single_table">
        <column name="single_id" type="numeric" />
        <column name="id" type="numeric" />
        <column name="status" type="varchar" />
    </metadata>
    <metadata data-nodes="expected_dataset.t_merchant">
        <column name="merchant_id" type="numeric" />
        <column name="country_id" type="numeric" />
        <column name="merchant_name" type="varchar" />
        <column name="business_code" type="varchar" />
        <column name="telephone" type="varchar" />
        <column name="creation_date" type="datetime" />
    </metadata>
    <row data-node="expected_dataset.t_single_table" values="1, 10, init" />
    <row data-node="expected_dataset.t_single_table" values="2, 11, init" />
    <row data-node="expected_dataset.t_single_table" values="3, 12, init" />
    <row data-node="expected_dataset.t_single_table" values="4, 13, init" />
    <row data-node="expected_dataset.t_user" values="10, 10000, a10, init" />
    <row data-node="expected_dataset.t_user" values="11, 11000, b11, init" />
    <row data-node="expected_dataset.t_user" values="12, 12000, c12, init" />
    <row data-node="expected_dataset.t_user" values="13, 13000, d13, init" />
    <row data-node="expected_dataset.t_user_item" values="100000, 10, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="110000, 11, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="120000, 12, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="130000, 13, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="140000, 14, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="150000, 15, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="160000, 16, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="170000, 17, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="180000, 18, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="190000, 19, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="200000, 20, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="210000, 21, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="220000, 22, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="230000, 23, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="240000, 24, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="250000, 25, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="260000, 26, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="270000, 27, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="280000, 28, init, 2017-08-08" />
    <row data-node="expected_dataset.t_user_item" values="290000, 29, init, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="1, 86, tencent, 86000001, 86100000001, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="2, 86, haier, 86000002, 86100000002, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="3, 86, huawei, 86000003, 86100000003, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="4, 86, alibaba, 86000004, 86100000004, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="5, 86, lenovo, 86000005, 86100000005, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="6, 86, moutai, 86000006, 86100000006, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="7, 86, baidu, 86000007, 86100000007, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="8, 86, xiaomi, 86000008, 86100000008, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="9, 86, vivo, 86000009, 86100000009, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="10, 86, oppo, 86000010, 86100000010, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="11, 1, google, 01000011, 01100000011, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="12, 1, walmart, 01000012, 01100000012, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="13, 1, amazon, 01000013, 01100000013, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="14, 1, apple, 01000014, 01100000014, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="15, 1, microsoft, 01000015, 01100000015, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="16, 1, dell, 01000016, 01100000016, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="17, 1, johnson, 01000017, 01100000017, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="18, 1, intel, 01000018, 01100000018, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="19, 1, hp, 01000019, 01100000019, 2017-08-08" />
    <row data-node="expected_dataset.t_merchant" values="20, 1, tesla, 01000020, 01100000020, 2017-08-08" />
</dataset>
